IIPS/iip-52.md

```diff
@@ -169,73 +169,70 @@ At this time, the `_from` would be the BTP address of `xcall`.
 
 ### Fees Handling
 
-The RI for ICON network has adopted a simple fixed fees handling mechanism. If a user wants to make a call from
-ICON to the T1 network, he needs to pay X ICX, and for the T2 network he needs to pay Y ICX.  That is, the fees
-depend on the destination network address.  Also, a `default` fee has been defined in case of there is no fee
-mapping to the network address.
+If a user wants to make a call from ICON to the T1 network, he needs to pay X ICX, and for the T2 network
```

Comment:
Please define `T1 network`. The first that it is mentioned here and in [the other IIP linked above](https://github.com/icon-project/IIPs/blob/master/IIPS/iip-25.md) is in this spot

For BTP-related blockchain networks, the standard so far is `source network` or `source blockchain` and `destination network` or `destination blockchain`

Reply:
Please define it the first time it is used. Further usages can be used in an abbreviated way. E.g.:

`...Target Network 1 (T1), he needs to pay X ICX, and for Target Network 2 (T2)...`

Reply:
@sink772  Could you describe more complex case when user wants to make a call from T1  to the T2 (T1 -> Icon -> T2)
How to calculate fee, in case of different relays? (T1 -> Icon : relay 1 |  Icon -> T2 : relay 2)

Reply:
@andrii-kl The relay fee handing will be described in iip-25 doc.  Recently @mksong-icon created a PR for it, see #58.